### ISSUE TITLE
Seemingly no tick function?

### ISSUE THREAD

**Issue by FracturedFlux** [monkeyhue/btl-smp-demo#2]
https://github.com/monkeyhue/btl-smp-demo/assets/131806616/aaff8d95-7cea-4d99-ad1a-a36fd14916e2

The datapack, when downloaded from this very github page, does not work.
Advencements and basic functions work, but nothing after that.

Just thought I'd mention/ask why.

**Comment by monkeyhue** [comment]
Because this is the raw github download, it does not come bundled with any of the smithed libraries required to run. Please download and use the following libraries in order for the pack to function properly:
- Smithed Actionbar (0.4.1+)
- Smithed Block (0.2.0+)
- Smithed Crafter (latest)

Once these are downloaded (make sure to merge with mito or something), do a reload and things should all work fine

**Comment by FracturedFlux** [comment]
Yeah, I knew I just misunderstood the readme, just couldnt figure out how.
